src/App.tsx

```diff
@@ -76,6 +77,10 @@ const PAGES = [
     label: TEXT_KEYS.about_title,
     key: '/about',
   },
+  // {
+  //   label: TEXT_KEYS.profile_page,
+  //   key: '/profile',
+  // },
```

Comment:
please remove,
you can add a different array for invisible pages

Reply:
conflicts were resolved